cookbook/bundles/best_practices.rst

```diff
@@ -174,7 +161,7 @@ Vendors
 A bundle must not embed third-party PHP libraries. It should rely on the
 standard Symfony autoloading instead.
 
-A bundle should not embed third-party libraries written in JavaScript, CSS, or
+A bundle should not embed third-party libraries written in JavaScript, CSS or
```

Comment:
In this recommendation we tell the developer what not to do, but we don't provide neither reasons nor alternatives. In the previous paragraph we say: "don't embed PHP libraries ... but use Symfony autoloading instead". Maybe we could add a short phrase like: "Use Grunt or XXX to manage those front-end dependencies."